Formula/baresip.rb

```diff
@@ -11,21 +12,30 @@ class Baresip < Formula
     sha256 "b99e262d153eb3414c2a6fe813be98e78f71da205d66ede0ec799d1e07f0341a" => :high_sierra
   end
 
-  depends_on "libre"
   depends_on "librem"
 
+  conflicts_with "libre", because: "both install `libre` lib"
+
+  resource "libre" do
+    url "https://github.com/baresip/re/archive/v1.1.0.tar.gz"
+    sha256 "82afc903a6b1e436b357f05eb11560ec681712291dd9c83c408465c895f28b5d"
+  end
+
   def install
     # baresip doesn't like the 10.11 SDK when on Yosemite
     if MacOS::Xcode.version.to_i >= 7
       ENV.delete("SDKROOT")
       ENV.delete("HOMEBREW_SDKROOT") if MacOS::Xcode.without_clt?
     end
```

Comment:
Is this still needed?

Reply:
I dont think we need that (`yosemite` hack 😅 )